client/plots/hierCluster.js

```diff
@@ -89,7 +90,7 @@ export class HierCluster extends Matrix {
 				// for now backend returns {gene:str, data:{}} if there's only 1 eligible gene
 				throw `Cannot do clustering: data is only available for 1 gene (${d.gene}). Try again by adding more genes.`
 			}
-			throw 'Cannot do clustering: invalid server response (lacks .clustering{})'
+			//throw 'Cannot do clustering: invalid server response (lacks .clustering{})'
```

Comment:
should this be reverted or deleted?

Reply:
if so please revert the change

Reply:
I meant it can not throw if there is no gene property. It is ok like it is.